kclvm/tools/src/LSP/src/util.rs

```diff
@@ -1,7 +1,24 @@
 use parking_lot::RwLockReadGuard;
+use std::{
+    ffi::OsString,
+    fs::read_dir,
+    io::{self, ErrorKind},
+    path::{Path, PathBuf},
+    vec,
+};
+
+use kclvm_cmd::settings::_build_settings;
+
+use kclvm_config::modfile::KCL_MOD_PATH_ENV;
+use kclvm_parser::LoadProgramOptions;
+use kclvm_runner::{join_work_path_with_k_file, ExecProgramArgs};
 use ra_ap_vfs::{FileId, Vfs};
 use serde::{de::DeserializeOwned, Serialize};
 
+pub static KCL_YAML: &'static str = "kcl.yaml";
+pub static CI_TEST_DIR: &'static str = "ci-test";
```

Comment:
Ci-test and settings are unnecessary because the compiled file will only take effect in one `kcl.yaml` file